2014-05-05  Aurelien Jarno  <aurelien@aurel32.net>

	* locales/it_IT: Use same grouping in LC_NUMERIC as in LC_MONETARY.

diff --git a/localedata/locales/it_IT b/localedata/locales/it_IT
--- a/localedata/locales/it_IT
+++ b/localedata/locales/it_IT
@@ -82,8 +82,8 @@ END LC_MONETARY
 
 LC_NUMERIC
 decimal_point             "<U002C>"
-thousands_sep             ""
-grouping                  0;0
+thousands_sep             "<U002E>"
+grouping                  3;3
 END LC_NUMERIC
 
 LC_TIME
